packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4369,67 +4369,343 @@ class PositionedDirectional extends StatelessWidget {
-///    not [Expanded]) with unbounded main axis constraints and the incoming
-///    cross axis constraints. If the [crossAxisAlignment] is
-///    [CrossAxisAlignment.stretch], instead use tight cross axis constraints
-///    that match the incoming max extent in the cross axis.
-/// 2. Divide the remaining main axis space among the children with non-zero
-///    flex factors (e.g., those that are [Expanded]) according to their flex
-///    factor. For example, a child with a flex factor of 2.0 will receive twice
-///    the amount of main axis space as a child with a flex factor of 1.0.
-/// 3. Layout each of the remaining children with the same cross axis
-///    constraints as in step 1, but instead of using unbounded main axis
-///    constraints, use max axis constraints based on the amount of space
-///    allocated in step 2. Children with [Flexible.fit] properties that are
-///    [FlexFit.tight] are given tight constraints (i.e., forced to fill the
-///    allocated space), and children with [Flexible.fit] properties that are
-///    [FlexFit.loose] are given loose constraints (i.e., not forced to fill the
-///    allocated space).
-/// 4. The cross axis extent of the [Flex] is the maximum cross axis extent of
-///    the children (which will always satisfy the incoming constraints).
-/// 5. The main axis extent of the [Flex] is determined by the [mainAxisSize]
-///    property. If the [mainAxisSize] property is [MainAxisSize.max], then the
-///    main axis extent of the [Flex] is the max extent of the incoming main
-///    axis constraints. If the [mainAxisSize] property is [MainAxisSize.min],
-///    then the main axis extent of the [Flex] is the sum of the main axis
-///    extents of the children (subject to the incoming constraints).
 /// 6. Determine the position for each child according to the
 ///    [mainAxisAlignment] and the [crossAxisAlignment]. For example, if the
 ///    [mainAxisAlignment] is [MainAxisAlignment.spaceBetween], any main axis
 ///    space that has not been allocated to children is divided evenly and
 ///    placed between the children.
 ///
+///    If the [crossAxisAlignment] is set to [CrossAxisAlignment.stretch],
+///    _all_ of the children will be forced to be as large as the parent of the
+///    [Flex] will allow, in the cross axis.
+///
+/// Advanced tip:
+/// What if you need to force the cross axis size of all
+/// children to match the size of the largest _child_ in the cross axis,
+/// but you do not want your [Flex] to be larger than that largest
+/// child, _even if the parent would allow it?_
+///
+///  You can use [CrossAxisAlignment.stretch] but then wrap
+///  a [Column] in an [IntrinsicWidth], or a [Row] in an
+///  [IntrinsicHeight]. The intrinsic tells the [Flex] that it does
+///  not have to be larger than its largest child (in the cross axis),
+///  but using [CrossAxisAlignment.stretch] tells the children
+///  to all be as big as the [Flex] will allow.
+///
+/// IE:
```

Comment:
```suggestion
/// For example:
```

And this should probably be inside of a ` ```dart  ` section.